pkgs/by-name/ap/apkeditor/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildEnv,
+  fetchurl,
+  jre,
+  writeShellScriptBin,
+}:
+
+let
+  pname = "apkeditor";
+  version = "1.4.1";
+
+  jar = fetchurl {
+    url = "https://github.com/REAndroid/APKEditor/releases/download/V${version}/APKEditor-${version}.jar";
+    hash = "sha256-SpiuanVSZDV2A/GQa5LmLnV9WSKQFOMzlfTgP/AZ/ag=";
+  };
+in
+buildEnv {
+  name = "${pname}-${version}";
+
+  paths = [
+    (writeShellScriptBin "APKEditor" ''
+      exec ${lib.getExe jre} -jar ${jar} "$@"
+    '')
+  ];
+
+  meta = {
```

Comment:
Missed this last round but since this is a downloaded binary, sourceProvenance should be set to binaryBytecode and optionally binaryNativeCode if it also contains native libraries.